guides/advanced_guides/known_limitations.md

```diff
@@ -0,0 +1,27 @@
+# Known limitations
+
+MeiliSearch has known limitations, some of which may be addressed in future versions
+
+## Design limitations
+
+### Database size
+
+MeiliSearch uses two database: one for storage and the second one for updates.
+Default database maximum size is 100GiB.
+MeiliSearch uses LMDB under the hood as the storage engine. On launch, LMDB need to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory, which is different from the real memory. You can read more [about storage](/resources/about_storage.md).
+You can modify this default limit using the options `--max-mdb-size` & `--max-udb-size` as described in the [configuration guide](/guides/advanced_guides/configuration.md#max-mdb-size).
+
+### Number of indexes
+
+You can create up to 200 indexes in MeiliSearch. When using LMDB, MeiliSearch have to specify a number of databases that must stay low for performance reasons.
+
+### Maximum words per attribute
+
+A maximum of 1000 words par attribute can be indexed. This limit is enforced for relevancy reasons. The more words there are in a given attribute, the less relevant the search queries will be.
```

Comment:
What do you mean by attribute ?
<img width="430" alt="Screenshot 2020-08-18 at 11 04 15" src="https://user-images.githubusercontent.com/33010418/90493360-9315ec00-e142-11ea-9ff3-836a0319b20b.png">

Do you mean the `key` or the value of a field?

Reply:
For clarity I think `1000 words per attribute` is more understandable. Or `1000 word per key`? I won't write `1000 words for a field's value` 😅
What do you think?

Reply:
I would suggest we change this part with the following suggestion

```
A field can contain a maximum of 1000 words. If you put more than 1000 words in one attribute, only the first 1000 will be indexed.

This limit is enforced for relevancy reasons as the more words there are in a given attribute, the less relevant the search queries will be.
```

Also, is this only for relevancy reasons or also for perfomance reasons ?

Reply:
I understand your point, I  would like @matboivin opinion on it. But, could you answer on the newer suggestion?